core-java-modules/core-java-arrays-guides/src/test/java/com/baeldung/print2DArrays/Print2DArraysTest.java

```diff
@@ -0,0 +1,54 @@
+package com.baeldung.print2DArrays;
```

Comment:
We follow this convention for Java code:
https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names

Please rename the package to be lowercase

Reply:
renamed